base/gcutils.jl

```diff
@@ -243,12 +243,21 @@ end
     GC.safepoint()
 
 Inserts a point in the program where garbage collection may run.
-This can be useful in rare cases in multi-threaded programs where some threads
-are allocating memory (and hence may need to run GC) but other threads are doing
-only simple operations (no allocation, task switches, or I/O).
-Calling this function periodically in non-allocating threads allows garbage
+
+Safepoint are fast, and do not themselves trigger garbage collection.
+However, if another task has requested the GC to run, reaching a safepoint will
+cause the current task to block and wait for the GC.
```

Comment:
Using `threads` here instead of `tasks` would be more accurate.

Reply:
Done